src/curated-corpus/components/SidebarWrapper/SidebarWrapper.styles.tsx

```diff
@@ -0,0 +1,18 @@
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+
+/**
+ * Styles for the SidebarWrapper component.
+ */
+export const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      background: 'white',
+      position: 'sticky',
+      top: 0,
+      bottom: 0,
+      paddingTop: '0.5rem',
+      paddingBottom: '0.5rem',
+      zIndex: 5,
```

Comment:
That's an oddly specific `z-index` 😄 do we have other DOM elements that have a z-index higher than 0 or 1?

Reply:
I don't think so, not on this page anyway. I think the `5` was lifted straight out of whatever StackOverflow example I was modifying to my use case - traditionally you'd set `z-index` to 100 or 1000 to make sure it's definitely on top of everything else on the page :)